scripts/instagrammer.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/zsh
+
+cd ~/documents/CSSA/
```

Comment:
will have to abstract this path away somehow, as it may be different for future maintainers